coderd/database/dbauthz/dbauthz.go

```diff
@@ -1992,6 +1992,10 @@ func (q *querier) GetLogoURL(ctx context.Context) (string, error) {
 	return q.db.GetLogoURL(ctx)
 }
 
+func (q *querier) GetNotStartedProvisionerJobs(ctx context.Context, notStartedSince time.Time) ([]database.ProvisionerJob, error) {
+	return q.db.GetNotStartedProvisionerJobs(ctx, notStartedSince)
+}
```

Comment:
The arg should probably be renamed to `lastUpdatedBefore` in all the places we do it. It's used in the query as `updated_at < $1`